### PR TITLE
Use NVIDIA Sans font in Transformer Engine documentation

### PR DESCRIPTION
**Pull request by ptrendx**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -85,7 +86,11 @@
 
 html_theme = 'sphinx_rtd_theme'
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-html_static_path = []
+html_static_path = ['_static']
```
Comment (ksivaman) [inline docs/conf.py:89]: Why is this needed? IIRC docs build fine without adding the `_static` but adding this results in a warning.

**Review by timmoon10 — APPROVED** [review]
lgtm_comic_sans

**Review by ksivaman — APPROVED** [review]
LGTM
